src/playback/qml/MuseScore/Playback/internal/TempoSlider.qml

```diff
@@ -52,7 +52,7 @@ RowLayout {
     }
 
     NumberInputField {
-        value: root.value * 100
+        value: (root.value * 100).toFixed(0)
```

Comment:
Shouldn't we rather use Math.round instead?

Reply:
Yeah right, we can use Math.round